### PR TITLE
Add content cards class / directives

### PR DESCRIPTION
**Pull request by simonpeters**

### CODE REVIEW

**Comment by simonpeters** [comment]
@VincentPeters

File: public/src/js/contentCards/directives/CouponCard.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('couponCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/couponCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/CouponCard.js]: excess whiteline

File: public/src/js/contentCards/directives/EventCard.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('eventCard', directive);
+
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/eventCard.html',
+
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/EventCard.js]: excess whiteline

File: public/src/js/contentCards/directives/NewsCard.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('newsCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/newsCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/NewsCard.js]: excess whiteline

File: public/src/js/contentCards/directives/OtherCard.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('otherCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/otherCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/OtherCard.js]: excess whiteline

File: public/src/js/contentCards/directives/PartnerCard.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('partnerCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/partnerCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/PartnerCard.js]: excess whiteline

File: public/src/js/contentCards/directives/PictureCard.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('pictureCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/pictureCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/PictureCard.js]: excess whiteline

File: public/src/js/contentCards/directives/PostBuzzInfoCard.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('postBuzzInfoCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/postBuzzInfoCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/PostBuzzInfoCard.js]: excess whiteline

File: public/src/js/contentCards/directives/PromotionCard.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('promotionCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/promotionCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/PromotionCard.js]: excess whiteline

File: public/src/js/contentCards/directives/SponsorCard.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('sponsorCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/sponsorCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/SponsorCard.js]: excess whiteline

File: public/src/js/contentCards/directives/advertCard.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('advertCard', directive);
+
+    function directive() {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/advertCard.html',
+
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/advertCard.js]: excess whiteline

File: public/src/js/contentCards/directives/aggregatedNewsCard.js [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+    angular.module('app.contentCards')
+        .directive('aggregatedNewsCard', directive);
+    
+    function directive () {
+        return {
+            restrict: 'E',
+            templateUrl: 'contentCards/directives/aggregatedNewsCard.html',
+            
+            controller($scope) {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/directives/aggregatedNewsCard.js]: excess whiteline

File: public/src/js/contentCards/module.js [hunk 12]
```diff
@@ -0,0 +1,11 @@
+(function () {
+    'use strict';
+    
+    angular
+        .module('app.contentCards', [])
+        .config(config);
+    
+    function config() {
+
```
Comment (VincentPeters) [inline public/src/js/contentCards/module.js]: excess whiteline

**Pull request merged**